themes/default/content/docs/reference/cli/environment-variables.md

```diff
@@ -155,10 +163,23 @@ meta_desc: A list of different environment variables the Pulumi CLI supports.
     </dt>
     <dd>
         <p>
-            Set this environment variable to opt-in to using `yarn` instead of `npm` for installing Node.js dependencies.
+            Set this environment variable to opt-in to using {{% md %}}`yarn`{{% /md %}} instead of {{% md %}}`npm`{{% /md %}} for installing Node.js dependencies.
```

Comment:
```suggestion
            Set this environment variable to opt-in to using <code>yarn</code> instead of <code>npm</code> for installing Node.js dependencies.
```